src/use-cases/createUseCase.ts

```diff
@@ -0,0 +1,47 @@
+import { hash } from 'bcryptjs'
+import { UserRepository } from '../repositories/user-repository'
+import { UserAlreadyExistsError } from './errors/user-already-exists-error'
+import { User } from '@prisma/client'
+
+interface CreateUseCaseRequest {
```

Comment:
What should we create? Can we name the class, the file, and the interfaces following the standard?

Suggestion: **CreateUserUseCase**